karate-core/src/main/java/com/intuit/karate/Actions.java

```diff
@@ -138,4 +139,6 @@
     
     void robot(String expression);
 
+    void lightHouseWrapper(String URL, List<String> options) throws IOException, InterruptedException;
```

Comment:
wrapper around node installed lighthouse module. This takes the URL to analyze and is configurable [ options ].

Reply:
for all driver stuff I prefer this as a `driver.lighthouse()` API

I know, it is a shift from how karate started, but something I think about. some thoughts here: https://github.com/intuit/karate/issues/398

Reply:
@ptrthomas - actually, this capability isn't linked to driver, hence I didn't placed it there. Lighthouse is cli based, and creates its own cdp session.